code/datums/flock/flock.dm

```diff
@@ -743,9 +761,15 @@ var/flock_signal_unleashed = FALSE
 						break
 				if (istype(O, /obj/machinery/light/lamp) || istype(O, /obj/machinery/computer3/generic/personal) || istype(O, /obj/machinery/computer3/luggable))
 					break
+				if (istype(O, /obj/machinery/vending) || istype(O, /obj/machinery/manufacturer) || istype(O, /obj/machinery/dispenser) || istype(O, /obj/machinery/disposal_pipedispenser) || istype(O, /obj/machinery/portable_atmospherics/canister) || istype(O, /obj/machinery/atmospherics/pipe/tank) || istype(O, /obj/machinery/portable_atmospherics/scrubber) || istype(O, /obj/machinery/chem_dispenser) || istype(O, /obj/machinery/chemicompiler_stationary))
```

Comment:
This makes me want to cry. Please no more lists of types that will need constant maintenance. Remember the atom prop stuff we did that got rid of all those?

Reply:
Not sure if there is any solution for this unless. Using atom properties still requires putting the object type in the conversion list, as well as adding atom properties to the subtypes that aren't converted, idk how appealing that is